cookbook/install/unstable_versions.rst

```diff
@@ -0,0 +1,82 @@
+How to Install and Use an Unstable Symfony Version
```

Comment:
I'd change this - I thought "unstable" at first meant an "old, unsupported" version (e.g. 2.4)". I know that's not logical, but that's what I thought :). How about:

>
How to Install or Upgrade to the Latest, Unreleased Symfony Version